cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -19,6 +19,9 @@ spec:
         version: v11
         kubernetes.io/cluster-service: "true"
     spec:
+{% if grains['cloud'] is defined and grains['cloud'] in [ 'vsphere', 'photon-controller' ] %}
```

Comment:
Hey guys, surprise this broke things.

#25020

There are a bunch of scripts that process this file and sed in the correct values.  They are all broken now.

I think the real fix is is find a way to configure kube-dns to know which nameserver(s) and searches to use, one option being "whatever my host is using".  What does Photon do in the normal docker case - do you demand a --dns flag for every container?  Do you set it on the daemon?

Kubelet has a --resolv-conf flag which lets you point it at an alternate file, I think that should handle this case for you?

In the meantime we need to either roll this PR back or find a fix that doesn't break all the scripts (and I don't think the answer is "fix the scripts").  Either use resolv-conf flag in your setting and get rid of this conditional or else fork this file or this dir into a photon-specific place.

ASAP please.